### ISSUE TITLE
1.10.5 segmentation fault on startup only on macOS 10.14

### ISSUE THREAD

**Issue by MarioG-X** [signalwire/freeswitch#810]
During startup on macOS 10.14, there is a segmentation fault:
2020-08-18 17:49:55.649640 [NOTICE] sofia.c:6195 Started Profile internal [sofia_reg_internal]
2020-08-18 17:49:55.655513 [WARNING] sofia.c:2306 MSG Thread 0 Started
Segmentation fault: 11

1.10.5 release works fine on macOS 10.15 and 10.13. 

Earlier in the day, 1.10.5 development worked fine on 10.14. After it was changed to release, both release and development get the error. A few days ago, this problem was only on 10.13, but now 10.13 is fine.

I erased /use/local, reinstalled CLT, rebooted, etc. twice to verify this was consistent.

I have attached the macOS crash report.
[freeswitch_2020-08-18-174958_FS-1014.txt](https://github.com/signalwire/freeswitch/files/5093523/freeswitch_2020-08-18-174958_FS-1014.txt)

**Comment by MarioG-X** [comment]
I decided to see if any changes to development in last 2 days would resolve this. Here is what happened today:

1. I deleted all of /usr/local and installed 1.10.6 development and everything from scratch, this is the third time since the start of failure 2 days ago.
2. This time, 1.10.6 came up! When I last tested development 2 days ago, it still showed 1.10.5 and got segmentation fault.
3. I downloaded and reinstalled release 1.10.5 expecting the segment fault. NO! It worked.

Don't know what is going on, problem was on macOS 10.13, then moved to 10.14, never on 10.15, then disappeared on 10.14 today. I will be dropping 10.13 testing soon. For 10.14, I will retest weekly to see if it pops up.

I wonder if homebrew could be involved. Today, I noticed it updated glib on 10.14.

I will give this a couple of weeks of retesting a full scratch and install a few times before closing.
